index.html

```diff
@@ -3121,6 +3127,14 @@ <h3 id="applying">
             prefix.
           </p>
           <p>
+            We encourage implementors to add proprietary extensions to our
+            <a href=
+            "https://github.com/w3c/manifest/wiki/Extensions-Registry">Extensions
+            Registry</a>. This allows the community to track what extension to
+            be made and periodically consider those extensions for
+            standardization.
```

Comment:
Something like the above would be great as part of the introduction of the wiki page.